### PR TITLE
Fix indiscernible text color

### PR DESCRIPTION
**Pull request by jadb**

### CODE REVIEW

File: webroot/css/cake.css [hunk 1]
```diff
@@ -16,7 +16,7 @@ a {
 }
 
 .side-nav li a:not(.button):hover {
-    color: #15848F;
+    color: #15848F;c
```
Comment (dereuromark) [inline webroot/css/cake.css]: typo?

Comment (jadb) [inline webroot/css/cake.css]: Arg! was a web commit - fixing now.

**Comment by lorenzo** [comment]
Can you post a screenshot of before and after?

**Comment by jadb** [comment]
screenshot

**Pull request merged**